head/src/main/kotlin/io/qalipsis/core/head/web/TimeSeriesController.kt

```diff
@@ -106,6 +106,12 @@ class TimeSeriesController(
             `in` = ParameterIn.QUERY
         )
         @Nullable @QueryValue("scenarios") scenarios: Set<@NotBlank String>?,
+        @Parameter(
+            description = "Zones to aggregate on; defaults to all scenarios of the specified campaigns.",
```

Comment:
The documentation is not fully correct. Please update for this and others like it.

> *defaults to all scenarios of the specified campaigns.